database/migration/aruha-312-set-default-retention-time-when-it-was-not-provided.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE zn_data.event_type
+SET et_event_type_object=jsonb_set(et_event_type_object, '{options}', '{"retention_time": 172800000}')
+WHERE et_event_type_object->>'options' = '{"retention_time": null}' OR et_event_type_object->'options' is null;
```

Comment:
Nice 👍 I think no one in the team has used postgres json manipulation properly before. Learned something.